packages/flutter/lib/src/material/text_selection.dart

```diff
@@ -84,22 +83,26 @@ class _TextSelectionToolbarState extends State<_TextSelectionToolbar> with Ticke
   Widget _getItem(_ItemData itemData, bool isFirst, bool isLast) {
     assert(isFirst != null);
     assert(isLast != null);
-    return ButtonTheme.fromButtonThemeData(
-      data: ButtonTheme.of(context).copyWith(
-        height: kMinInteractiveDimension,
-        minWidth: kMinInteractiveDimension,
-      ),
-      child: FlatButton(
-        onPressed: itemData.onPressed,
+
+    // TODO(hansmuller): Should be colorScheme.onSurface
```

Comment:
Just pointing out this TODO in case you forgot it, otherwise I assume it's for another PR.

Reply:
I'll open an issue. The onSurface color causes a tiny difference, black instead of "black87", which causes some golden image tests to fail. Will need to do a separate update to change the color.